deployment/ansible/roles/raster-foundry.base/defaults/main.yml

```diff
@@ -9,3 +9,5 @@ envdir_config:
   RF_STATSD_HOST: "{{ statsite_host }}"
   RF_CACHE_HOST: "{{ redis_host }}"
   RF_CACHE_PORT: "{{ redis_port }}"
+  RF_AWS_PROFILE: "{{ aws_profile }}"
```

Comment:
If this is `RF_AWS_PROFILE` vs. `AWS_PROFILE`, Boto won't be able to look it up automatically.